ci-operator/config/openshift/windows-machine-config-operator/openshift-windows-machine-config-operator-community-4.10__okd.yaml

```diff
@@ -0,0 +1,62 @@
+base_images:
+  ocp_builder_rhel-8-golang-1.17-openshift-4.10:
+    name: builder
+    namespace: ocp
+    tag: rhel-8-golang-1.17-openshift-4.10
+binary_build_commands: make build
+build_root:
+  from_repository: true
+  use_build_cache: true
+images:
+- dockerfile_path: build/Dockerfile.ci
+  inputs:
+    ocp_builder_rhel-8-golang-1.17-openshift-4.10:
+      as:
+      - registry.ci.openshift.org/ocp/builder:rhel-8-golang-1.17-openshift-4.10
+  to: windows-machine-config-operator-test
+resources:
+  '*':
+    limits:
+      memory: 4Gi
+    requests:
+      cpu: 100m
+      memory: 400Mi
+  src:
+    limits:
+      memory: 10Gi
+    requests:
+      cpu: 100m
+      memory: 10Gi
+  windows-machine-config-operator-test:
+    limits:
+      memory: 10Gi
+    requests:
+      cpu: 100m
+      memory: 10Gi
+tag_specification:
```

Comment:
is tag_specification not in use anymore? What's the benefit of using releases instead?

Reply:
[Inject the oc cli](https://docs.ci.openshift.org/docs/architecture/step-registry/#injecting-the-oc-cli), for instance.

Reply:
since [non-community 4.10 uses releases](https://github.com/openshift/release/blob/ad474217fc37f5fa650d5bf06916d981bf4ff957/ci-operator/config/openshift/windows-machine-config-operator/openshift-windows-machine-config-operator-release-4.10.yaml#L27) I agree that this PR should also use it